packages/shared/src/modules/ai-assistant/libs/enums/ai-assistant-api-path.enum.ts

```diff
@@ -0,0 +1,11 @@
+const AiAssistantApiPath = {
+	ACCEPT_TASK: "/chat/accept-task",
+	ADD_MESSAGE: "/chat/add-message",
+	CHANGE_TASK: "/chat/change-task",
+	CONTINUE_CHAT: "/chat/continue",
+	DELETE_CHAT: "/chat/remove",
```

Comment:
Why do we need DELETE_CHAT?

Reply:
The first idea is when we close chat, we delete chat.